src/integrationTest/java/uk/gov/hmcts/reform/professionalapi/FindUsersByOrganisationIntegrationTest.java

```diff
@@ -257,35 +257,42 @@ private void validateUsers(Map<String, Object> response, int expectedUserCount,
     @Test
     public void can_retrieve_users_when_false_should_return_status_200_without_roles() {
         Map<String, Object> response = professionalReferenceDataClient.findUsersByOrganisation(
-                createAndActivateOrganisation(), "True", hmctsAdmin,"false");
+                createAndActivateOrganisation(), "True", hmctsAdmin, "false");
         validateUsers(response, 3, false);
     }
 
     @Test
     public void can_retrieve_users_when_true_should_return_status_200_with_roles() {
         Map<String, Object> response = professionalReferenceDataClient.findUsersByOrganisation(
-                createAndActivateOrganisation(), "True", hmctsAdmin,"true");
+                createAndActivateOrganisation(), "True", hmctsAdmin, "true");
         validateUsers(response, 3, true);
     }
 
     @Test
     public void can_retrieve_users_when_default_should_return_status_200_with_roles() {
         Map<String, Object> response = professionalReferenceDataClient.findUsersByOrganisation(
-                createAndActivateOrganisation(), "True", hmctsAdmin,null);
+                createAndActivateOrganisation(), "True", hmctsAdmin, null);
         validateUsers(response, 3, true);
     }
 
+    @Test
+    public void retrieve_active_users_for_an_organisation_with_invalid_bearer_token_should_return_403() {
+        Map<String, Object> response = professionalReferenceDataClient.findUsersByOrganisationWithoutAuthHeaders(
+                createAndActivateOrganisation(), "True", null);
```

Comment:
1. True should be a boolean. This can be a design change outside the scope of this ticket.
2. If point 1 is not possible, "True" should be in lowercase to mimic a boolean rather than a String.

Reply:
1. It originally was, but I recall a decision being made to make it string rather than boolean
2. the casing is irrelevant as the endpoint carries out `.equalsIgnoreCase()` on the parameter

Reply:
Okay, makes sense.